content/en/blog/_posts/data-tbd/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: blog
+title: "Dynamic Resource Allocation in Kubernetes 1.26"
+date: 
+slug: dynamic-resource-allocation
+---
+
+ **Authors:** Patrick Ohly (Intel), Kevin Klues (NVIDIA)
+
+Dynamic resource allocation is a new API for requesting and sharing resources
+between pods and containers inside a pod. It is a generalization of the
+persistent volumes API for generic resources. Third-party resource drivers are
+responsible for tracking and allocating resources. Different kinds of
+resources support arbitrary parameters for defining requirements and
+initialization.
+
```

Comment:
Sharing between pods is already in the first sentence, and the other is implied by "arbitrary parameters". I would need to remove that to avoid repeating myself... yes, let's do that. It becomes a bit more explicit.